solidity/contracts/bridge/Bridge.sol

```diff
@@ -132,18 +216,71 @@ contract Bridge is Ownable {
     ///         keccak256(txHash | txOutputIndex | txOutputValue). The `tx`
     ///         prefix refers to the transaction which created that main UTXO.
+    ///         The txHash is bytes32 (ordered as in Bitcoin internally),
+    ///         txOutputIndex an uint32, and txOutputValue an uint64 value.
     mapping(bytes20 => bytes32) public mainUtxos;
 
+    /// @notice Collection of all pending redemption requests indexed by
+    ///         redemption key built as
+    ///         keccak256(walletPubKeyHash | redeemerOutputHash). The
+    ///         walletPubKeyHash is the 20-byte wallet's public key hash
+    ///         (computed using Bitcoin HASH160 over the compressed ECDSA
+    ///         public key) and redeemerOutputHash is the 20-byte (P2PKH,
+    ///         P2WPKH or P2SH) or 32-byte (P2WSH) output hash that will be
+    ///         used to lock redeemed BTC as requested by the redeemer.
+    ///         Requests are added to this mapping by the `requestRedemption`
+    ///         method (duplicates not allowed) and are removed by one of the
+    ///         following methods:
+    ///         - `submitRedemptionProof` in case the request was handled
+    ///           successfully
+    ///         - `notifyRedemptionTimeout` in case the request was reported
+    ///           to be timed out
+    ///         - `submitRedemptionFraudProof` in case the request was handled
+    ///           in an incorrect way amount-wise.
+    mapping(uint256 => RedemptionRequest) public pendingRedemptions;
+
+    /// @notice Collection of all invalid redemptions indexed by redemption key
+    ///         built as keccak256(walletPubKeyHash | redeemerOutputHash). The
+    ///         walletPubKeyHash is the 20-byte wallet's public key hash
+    ///         (computed using Bitcoin HASH160 over the compressed ECDSA
+    ///         public key) and redeemerOutputHash is the 20-byte (P2PKH,
+    ///         P2WPKH or P2SH) or 32-byte (P2WSH) output hash that is involved
+    ///         in the invalid redemption. Invalid redemptions are stored in
+    ///         this mapping to avoid slashing the wallets multiple times for
+    ///         the same fault. Two methods can add to this mapping:
+    ///         - `notifyRedemptionTimeout` which puts the redemption key
+    ///           to this mapping basing on a timed out request stored
+    ///           previously in `pendingRedemptions` mapping.
+    ///         - `submitRedemptionFraudProof` which puts the redemption key
+    ///           to this mapping basing on a misfunded request stored
+    ///           previously in `pendingRedemptions` mapping or
+    ///           basing on a non-requested arbitrary redemption performed
+    ///           by the wallet.
+    ///
+    // TODO: Remove that Slither disable once this variable is used.
+    // slither-disable-next-line uninitialized-state
+    mapping(uint256 => bool) public invalidRedemptions;
```

Comment:
I think this variable is already used, so perhaps we can remove the Slither diable line.